test/ClangModules/enum-objc.swift

```diff
@@ -0,0 +1,12 @@
+// FIXME: @objc(Renamed) enums should work
+// XFAIL: *
+
+// RUN: %target-swift-frontend -emit-sil %s -import-objc-header %S/Inputs/enum-objc.h -verify
```

Comment:
I wasn't sure if this sort of test is the right way to do this, but I was using the `test/ClangModules/enum-new.swift` test as a prototype.